TOOLS/lua/autoload.lua

```diff
@@ -16,6 +16,9 @@ disabled=no
 images=no
 videos=yes
 audio=yes
+extend_images=list,of,ext
+extend_videos=list,of,ext
+extend_audio=list,of,ext
```

Comment:
I know it's longer but how about `additional_image_exts` and so on just to be clearer?

Reply:
Changed